vector-im/element-android
# #5567: [Location Sharing] - Set duration of live sharing

## Type of change


## Content


Adding a new bottom sheet to ask user to choose between different maximum duration for the share. It is displayed before starting the live location sharing.

## Motivation and context


Closes #5567 

## Screenshots / GIFs


<img src="https://user-images.githubusercontent.com/46314705/161008090-553fe9cb-6383-43d6-bbf5-05efa1093a7a.png" width=300/><img src="https://user-images.githubusercontent.com/46314705/161008108-85fba17e-aeff-4d52-a384-e38437b334fe.png" width=300/>

## Tests



- Ensure location sharing is enabled in "Settings -> Preferences"
- Go to a room
- Select location sharing by pressing the add button and then the locate icon
- Press live location sharing option
- Check a dismissable bottom sheet is displayed with the different duration options
- Select one of the durations
- Press the share button
- Check the location sharing screen is closed
- Check the live location service is started

## Tested devices

- OS version(s): Android 11

## Checklist
